### PR TITLE
Auxiliary pages

### PR DESCRIPTION
**Pull request by mannyakosah**
# Description

> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Checklist

- [X] This PR can be reviewed in under 30 minutes
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [] I have assigned reviewers to this PR.

### Navigate to auth pages
navigate-to-other-pages

**Sign up : Existing user error**
existing-user-error

**Sign up**
sign-up-redirect

### Sign in : (works fine. not broken by new features)
sign-in

### Forgot password
forgot-password

### Reset password
reset-password

### Unknown server error message
unknown-server-error-message

### CODE REVIEW

**Review by mannyakosah — COMMENTED** [review]
i had to install webpack to in order to successfully run locally

**Review by cbolles — DISMISSED** [review]

File: packages/client/.env.local [hunk 1]
```diff
@@ -1 +1 @@
-VITE_AUTH_URL=https://test-auth-service.sail.codes
```
Comment (gregfrasco) [inline packages/client/.env.local:1]: Revert Change

File: packages/client/src/pages/sign-up.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import { TextInput } from '../components/forms/text-input';
+import { PasswordInput } from '../components/forms/password-input';
+import { SubmitButton } from '../components/forms/submit-button';
+import { useSignUpEmailMutation } from '../graphql/auth/auth';
+
+import { useEffect, useState } from 'react';
+import { useProject } from '../context/project.context';
+import { useNavigate } from 'react-router-dom';
+
+const LoginValidation = Yup.object().shape({
```
Comment (gregfrasco) [inline packages/client/src/pages/sign-up.tsx]: fix name

File: packages/server/package.json [hunk 3]
```diff
@@ -59,7 +59,8 @@
     "randomstring": "^1.2.3",
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.2",
-    "rxjs": "^7.2.0"
+    "rxjs": "^7.2.0",
+    "webpack": "^5.75.0"
```
Comment (gregfrasco) [inline packages/server/package.json]: add as dev dependency

File: packages/client/src/pages/forgot-password.tsx [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+});
+
+export const ForgotPassword = () => {
+  const [forgotPassword, { data, error }] = useForgotPasswordMutation();
+
+  const [errorText, setErrorText] = useState('');
+  const { project } = useProject();
+  const navigate = useNavigate();
+
+  useEffect(() => {
```
Comment (gregfrasco) [inline packages/client/src/pages/forgot-password.tsx:23]: remove for now if not used

**Review by gregfrasco — COMMENTED** [review]
Minor fixes, nice work!

File: packages/client/.env.local (hunk 1 above)
Comment (mannyakosah) [reply packages/client/.env.local:1]: reverted

File: packages/client/src/pages/forgot-password.tsx (hunk 4 above)
Comment (mannyakosah) [reply packages/client/src/pages/forgot-password.tsx:23]: removed

File: packages/client/src/pages/sign-up.tsx (hunk 2 above)
Comment (mannyakosah) [reply packages/client/src/pages/sign-up.tsx]: fixed

File: packages/server/package.json (hunk 3 above)
Comment (mannyakosah) [reply packages/server/package.json]: webpack added as dev dependency

Comment (gregfrasco) [reply packages/server/package.json]: Looks like it's still here

Comment (mannyakosah) [reply packages/server/package.json]: right. should be fixed now.

**Review by cbolles — APPROVED** [review]

**Review by gregfrasco — APPROVED** [review]

**Pull request merged**